archive.istio.io/public/index.html

```diff
@@ -80,7 +78,7 @@ <h1>istio.io archives</h1>
                                     }
                                     var li = e.appendChild(document.createElement("li"));
                                     var a = li.appendChild(document.createElement("a"));
-                                    a.setAttribute("href", "https://archive.istio.io/" + lines[i] + "/");
+                                    a.setAttribute("href", "./" + lines[i] + "/index.html");
```

Comment:
never put index.html when / works

Reply:
I did this change and the others to make the local site built with build.sh work. I thought this would also work on archive.istio.io proper so then I'd have something I could test locally before pushing. 

I've reverted these changes.